example/test.py

```diff
@@ -379,17 +439,24 @@ def test_gzip_small_file_not_compressed(self, web_container, web_client):
         ))
 
 
-@pytest.fixture(scope='class')
-def worker_container(docker_helper, amqp_container):
-    container = create_django_bootstrap_container(
-        docker_helper, 'worker', command=['celery', 'worker'],
-        publish_port=False)
-    docker_helper.start_container(container, r'celery@\w+ ready')
-    yield container
-    docker_helper.stop_and_remove_container(container)
+if SINGLE_CONTAINER:
+    @pytest.fixture(scope='class')
+    def worker_container(docker_helper, web_container):
+        docker_helper.start_container(web_container, r'celery@\w+ ready')
```

Comment:
Docker doesn't seem to care about trying to start containers that are already started. We do care about this log line, though.